torch/csrc/jit/tensorexpr/ir_simplifier.cpp

```diff
@@ -1921,15 +1921,15 @@ const Expr* simplifyRoundModPattern(const Polynomial* poly) {
   while (!mods.empty() && repeat) {
     repeat = false;
     // NOLINTNEXTLINE(bugprone-narrowing-conversions,cppcoreguidelines-narrowing-conversions)
-    for (int i = mods.size() - 1; i >= 0; i--) {
+    for (int64_t i = mods.size() - 1; i >= 0; i--) {
       const Term* m = mods[i];
       const Mod* mod = dynamic_cast<const Mod*>(m->variables()[0]);
       CHECK(mod);
       const Expr* mod_lhs = IRSimplifier::simplify(mod->lhs());
       const Expr* mod_rhs = IRSimplifier::simplify(mod->rhs());
       bool merged = false;
       // NOLINTNEXTLINE(bugprone-narrowing-conversions,cppcoreguidelines-narrowing-conversions)
```

Comment:
Comments can be removed in multiple places in this file no?

Reply:
This still makes a conversion from `size_t` to `int64_t`, which is narrowing. It's just not as bad now.